### ISSUE TITLE
Incorrect (or obsolete) project name in NBitcoin.sln

### ISSUE THREAD

**Issue by DeathAndTaxes** [NicolasDorier/NBitcoin#65]
Visual studio complains about being unable to find 'Build.csproj' when building.

Form NBitcoin.sln
```
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Build", "Build\Build.csproj", "{FCD2502B-440B-4C39-8DD5-1052DEBBB0E3}"
EndProject
```

There is a Build\Deploy.csproj.  I was unsure if solution should be pointed to Build\Deploy.csproj or just delete the project and reference in the solution.  I did the later and it built fine.

**Comment by NicolasDorier** [comment]
Can you tell me the error you got ?
Build.csproj is not really important for others, I'm only using it for pushing my stuff to nugget more easily.

**Comment by NicolasDorier** [comment]
Interesting, my visual studio was lying to me by making Build.csproj committed. I fixed that 663551273cf63baf6aca7ba72c5573d281cc2570
This project is not important for other people anyway, it is only a helper project for pushing on nuget.
